RoadMap_SwiftUI/RoadMap_SwiftUI/ViewModel/ClimateViewModel.swift

```diff
@@ -0,0 +1,33 @@
+// ClimateViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана климат контроля
+final class ClimateViewModel: ObservableObject {
+    // MARK: - Public properties
+
+    @Published var isAcOn = false
+    @Published var isPowerTapped = false
+    @Published var selectedColor: Color = .topGradient
+    @Published var acCircleProgress: CGFloat = 0.0
```

Comment:
```suggestion
    @Published var acCircleProgress: CGFloat = 0.0
```
переменные одного типа располагай рядом
```suggestion
    @Published var acCircleProgress: CGFloat = 0.0
    @Published var isAcOn = false
    @Published var isPowerTapped = false
    @Published var isDesclosureGroupExpanded = true
    @Published var isAlertShown = false
```